.vscode/tasks.json

```diff
@@ -0,0 +1,23 @@
+{
+    "tasks": [
+        {
+            "type": "cppbuild",
+            "label": "build cloe",
```

Comment:
How are tasks normally written? I would imagine `Build Cloe (without tests)` is a better label.